_posts/2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: post
+title: Overview and Logs for the Kovri Dev Meeting Held on 2018-06-07
```

Comment:
Other Kovri logs are like:
`title: Logs for the Kovri Dev Meeting Held on 2016-06-05`
This "Overview" is only used for monero dev meeting actually.
Whether i don't really care, at least, the file name should be relevant. As your filename is `2018-06-07-logs-for-the-kovri-dev-meeting-held-on-2018-06-07.md` i think the tilte should be `title: Logs for the Kovri Dev Meeting Held on 2018-06-07`

Reply:
humm... obviously it has derived over time. Lots of blog title are not in line with their file name. Let's forget about my comment then.